tests/e2e/release/standard.js

```diff
@@ -1,13 +1,59 @@
 const helper = require('../../helper');
+const utils = require('../../utils');
 const commonElements = require('../../page-objects/common/common.po.js');
 const contactPage = require('../../page-objects/contacts/contacts.po.js');
+const uuid = require('uuid');
+const districtId = uuid.v4();
 
 describe('Creating contacts with standard config', function() {
+  beforeAll(done => {
+    return Promise.all([
+      utils.saveDocs(expectedDocs)
+    ])
+    .then(() => done()).catch(done.fail);
+  });
+  
+  const expectedDocs = [
+    {
+      _id: districtId,
+      name: 'ADistrict',
+      type: 'district_hospital',
+      reported_date: Date.now(),
+    },
+    {
+      _id: uuid.v4,
+      parent: {
+        _id: districtId
+      },
+      name: 'AHealthCenter',
+      type: 'health_center',
+      reported_date: Date.now(),
+    }
+  ];
+  
   it('should create a new district hospital', async function() {
     const districtName = 'Test District';
     await commonElements.goToPeople();
     await contactPage.addNewDistrict(districtName);
     helper.waitUntilReady(contactPage.contactName);
     expect(contactPage.contactName.getText()).toBe(districtName);
   });
-});
\ No newline at end of file
+
+  it('should create a new health center', async function(){
+    await commonElements.goToPeople();
+    contactPage.selectLHSRowByText(expectedDocs[0].name);
```

Comment:
How confident are you that there is exactly one place with this name? It would be safer to use the UUID, or to generate a random name so the test doesn't fail unexpectedly.